src/identifiers.md

```diff
@@ -60,4 +63,37 @@ See the [HIR chapter][hir-map] for more detailed information.
 
 ## In the MIR
 
-**TODO**
+- [`BasicBlock`] identifies a *basic block*. It points to an instance of
+  [`BasicBlockData`].
+
+- [`Local`] identifies a local variable in a function. Its associated data is in
+  [`LocalDecl`].
+
+- [`Field`] identifies a struct's, union's, or enum variant's field. It is used
+  as a "projection" in [`Place`].
+
+- [`SourceScope`] identifies a name scope in the original source code. Used for
+  diagnostics and for debuginfo in debuggers.
```

Comment:
We should probably also mention `SourceScopeData` and how to get it here?

Reply:
Yes, thanks for catching that!